components/network/vpnc/Makefile

```diff
@@ -16,20 +16,25 @@ include ../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		vpnc
 COMPONENT_VERSION=	0.5.3
-COMPONENT_REVISION=	1
+COMPONENT_REVISION=	0
+HUMAN_VERSION = $(COMPONENT_VERSION).r550
 COMPONENT_ARCHIVE=	$(COMPONENT_NAME)-$(COMPONENT_VERSION).tar.gz
 COMPONENT_FMRI=		network/$(COMPONENT_NAME)
 COMPONENT_CLASSIFICATION=	Applications/Internet
-COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+# COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
 COMPONENT_PROJECT_URL=	https://www.unix-ag.uni-kl.de/~massar/vpnc/
-COMPONENT_ARCHIVE_URL=	https://www.unix-ag.uni-kl.de/~massar/vpnc/$(COMPONENT_ARCHIVE)
-COMPONENT_ARCHIVE_HASH=	\
+# COMPONENT_ARCHIVE_URL=	https://www.unix-ag.uni-kl.de/~massar/vpnc/$(COMPONENT_ARCHIVE)
+# COMPONENT_ARCHIVE_HASH=	\
 	sha256:46cea3bd02f207c62c7c6f2f22133382602baeda1dc320747809e94881414884
 COMPONENT_LICENSE=		GPLv2,BSD-2clause
 COMPONENT_LICENSE_FILE= COPYING
 COMPONENT_SUMMARY=	An IPSec-like VPN client compatible with Cisco EasyVPN
 COMPONENT_DESCRIPTION=	An IPSec-like (over TUN/TAP) VPN client compatible with Cisco 3000 VPN Concentrators and EasyVPN
 
+SVN_REPO = http://svn.unix-ag.uni-kl.de/vpnc/trunk
+SVN_REV = 550
+SVN_HASH = NOHASH
```

Comment:
Shouldn't it be SVN_HASH= for proper prep-svn.mk work?
Also note that hash should be populated before merging (we can upload tarball to server's USERLAND_ARCHIVES)